Mathlib/Data/Fin/Basic.lean

```diff
@@ -596,15 +600,18 @@ theorem cast_eq_cast (h : n = m) : (Fin.cast h : Fin n → Fin m) = _root_.cast
 
 /-- `Fin.castAdd` as an `Embedding`, `castAddEmb m i` embeds `i : Fin n` in `Fin (n+m)`.
 See also `Fin.natAddEmb` and `Fin.addNatEmb`. -/
-@[simps! apply]
 def castAddEmb (m) : Fin n ↪ Fin (n + m) := castLEEmb (le_add_right n m)
 
+@[simp]
+lemma castAddEmb_apply (m) : (castAddEmb m : Fin n → Fin (n + m)) = castAdd m := rfl
```

Comment:
Since we have `coe_succEmb` and `coe_castLEEmb`, it looks to me that `coe_castAddEmb : _ = castAdd` should exist too (as well as the `_apply` one you have here)